Jenkinsfile

```diff
@@ -73,9 +75,10 @@ pipeline {
 		stage('Post: Backup gk_central after modifications'){
 			steps{
 				script{
+				    def timestamp = new Date().format("yyyy-MM-dd-HHmmss")
```

Comment:
I noticed this same implementation for timestamp was used in the ChEBI update Jenkins file - could a function be created, put into a util module and imported wherever needed by Jenkins files in the release process (similar to my comment about the release number in the ChEBI update PR - https://github.com/reactome/release-chebi-update/pull/3#discussion_r503067985.